src/content/docs/ja/reference/configuration.mdx

```diff
@@ -0,0 +1,1011 @@
+      "allowComments": true
+    }
+  }
+}
+```
+
+### `json.parser.allowTrailingCommas`
+
+JSONファイル内の末尾カンマの解析を有効にします。
+
+
+
+```json title="biome.json"
+{
+  "json": {
+    "parser": {
+      "allowTrailingCommas": true
+    }
+  }
+}
+```
+
+### `json.formatter.enabled`
+
+JSON（およびその拡張言語）ファイル用のBiomeformatterを有効にします。
+
+> デフォルト: `true`
+
+### `json.formatter.indentStyle`
+
+JSON（およびその拡張言語）ファイルのインデントスタイルです。 `"tab"` または `"space"` のいずれかを指定できます。
+
+> デフォルト: `"tab"`
+
+
+### `json.formatter.indentSize`
+
+このオプションは廃止されました。代わりに [`json.formatter.indentWidth`](#jsonformatterindentwidth) を使用してください。
+
+<details>
+<summary>廃止されました</summary>
+
+JSON（およびその拡張言語）ファイルのインデントサイズを指定します。
+
+> デフォルト: `2`
+
+</details>
+
+### `json.formatter.indentWidth`
+
+JSON（およびその拡張言語）ファイルのインデントサイズを指定します。
+
+> デフォルト: `2`
+
+### `json.formatter.lineEnding`
+
+JSON（およびその拡張言語）ファイルの改行コードの種類を指定します。
+- `"lf"`: Line Feed のみ (`\n`)、Linux や macOS、または git リポジトリ内で一般的です。
+- `"crlf"`: Carriage Return + Line Feed 文字 (`\r\n`)、Windows で一般的です。
+- `"cr"`: Carriage Return 文字のみ (`\r`)、非常にまれに使用されます。
+
+> デフォルト: `"lf"`
+
+### `json.formatter.lineWidth`
+
+JSON（およびその拡張言語）ファイルの行の文字数を指定します。
+
+> デフォルト: `80`
+
+### `json.formatter.trailingCommas`
+
+複数行のカンマ区切りの構文構造で、可能な限り末尾カンマを追加します。
+
+許可される値は次の通りです。
+- ` "none"`: 末尾カンマは削除されます。
+- `"all"`: 末尾カンマは保持され、**推奨**されます。
+
+> デフォルト: `"none"`
+
+### `json.linter.enabled`
+
+JSON（およびその拡張言語）ファイル用のBiomeのlinterを有効にします。
+
+> デフォルト: `true`
+
+
+## `css`
+
+CSSファイルに適用されるオプションです。
+
+### `css.parser.cssModules`
+
+[CSS modules](https://github.com/css-modules/css-modules) の解析を有効にします。
+
+> デフォルト: `false`
+
+### `css.formatter.enabled`
+
+CSS（およびその拡張言語）ファイル用のBiomeのformatterを有効にします。
+
+> デフォルト: `false`
+
+### `css.formatter.indentStyle`
+
+CSS（およびその拡張言語）ファイルのインデントスタイルです。 `"tab"` または `"space"` のいずれかを指定できます。
+
+> デフォルト: `"tab"`
+
+
+### `css.formatter.indentWidth`
+
+CSS（およびその拡張言語）ファイルのインデントサイズを指定します。
+
+> デフォルト: `2`
+
+### `css.formatter.lineEnding`
+
+CSS（およびその拡張言語）ファイルの改行コードの種類を指定します。
+- `"lf"`: Line Feed のみ (`\n`)、Linux や macOS、または git リポジトリ内で一般的です。
+- `"crlf"`: Carriage Return + Line Feed 文字 (`\r\n`)、Windows で一般的です。
+- `"cr"`: Carriage Return 文字のみ (`\r`)、非常にまれに使用されます。
+
+> デフォルト: `"lf"`
+
+### `css.formatter.lineWidth`
+
+CSS（およびその拡張言語）ファイルの行の文字数を指定します。
+
+> デフォルト: `80`
+
+### `css.formatter.quoteStyle`
+
+文字列リテラルを表現する際に使用する引用符の種類です。 `"single"` または `"double"` のいずれかを指定できます。
+
+> デフォルト: `"double"`
+
+### `css.linter.enabled`
+
+CSS（およびその拡張言語）ファイル用のBiomeのlinterを有効にします。
+
+> デフォルト: `false`
+
+## `overrides`
+
+パターンのリストです。
+この設定を使用して、特定のファイルに対するツールの動作を変更します。
+ファイルがオーバーライドパターンに一致すると、そのパターンに指定された設定がトップレベルの設定を上書きします。
+パターンの順序は重要です。ファイルが3つのパターンに一致する場合、最初のパターンのみが使用されます。
+
+### `overrides.<ITEM>.ignore`
+
+Unixシェルスタイルのパターンのリストです。Biomeは、これらのパターンに一致するファイルに対してオーバーライドを適用しません。
+
+
+```json title="biome.json"
+{
+  "overrides": [{
+    "ignore": ["scripts/*.js"]
+  }]
+}
+```
+
+### `overrides.<ITEM>.include`
+
+Unixシェルスタイルのパターンのリストです。Biomeは、これらのパターンに一致するファイルにのみオーバーライドを適用します。
```

Comment:
same

Reply:
I fixed that. [c79413e](https://github.com/biomejs/website/pull/917/commits/c79413e54c5a9d7a7defc464356a5dcc12249eb3)